scripts/image-builder/configure.sh

```diff
@@ -23,6 +23,13 @@ fi
      createrepo yum-utils selinux-policy-devel jq wget lorax rpm-build \
      containernetworking-plugins expect"
 
+VID=$(source /etc/os-release && echo "${VERSION_ID}")
+if { [[ "${VID}" == "9.4" ]] && grep -qE 'Red Hat Enterprise Linux.*Beta' /etc/redhat-release; }; then
```

Comment:
I know we're not going to see a RHEL 9.5 for a minute, but could this be generalized so it's not something we have to continually update?  Maybe we could detect if it's a beta version?

Reply:
Yes, we should probably change it to be more generic, something like the following
```
if grep -qE 'Red Hat Enterprise Linux.*Beta' /etc/redhat-release; then
   VID=$(source /etc/os-release && echo "${VERSION_ID}")
   sudo sed -i "s,dist/rhel9/${VID},beta/rhel9/9,g" /usr/share/osbuild-composer/repositories/rhel-${VID}.json
fi
```